src/stateless/slider.vue

```diff
@@ -0,0 +1,546 @@
+<template>
+    <div :class="[theme, size] | prefix('slider--')" class="slider">
+        <div class="slider__row">
+            <div class="slider__label">
+                {{ label }}
+            </div>
+        </div>
+
+        <div class="slider__row">
+            <div class="slider__input">
+                <input-element
+                    :type="isWholeNumber ? 'number' : 'float'"
+                    :size="size"
+                    :disabled="disabled"
+                    :value="value"
+                    :step="step"
+                    :min-value="min"
+                    :max-value="max"
+                    :is-valid="isValidInput"
+                    :theme="theme"
+                    :alignment="alignment"
+                    :locale="locale"
+                    @input="handleInput"
+                ><span v-if="unit" slot="right">{{ unit }}</span></input-element>
+            </div>
+
+            <div ref="bar" :class="stateClass | prefix('bar--')" class="bar" tabindex="0" @mousedown="startDrag" @keydown.left.stop="decreaseValue" @keydown.down.stop="decreaseValue" @keydown.right.stop="increaseValue" @keydown.up.stop="increaseValue">
+                <div class="bar__container">
+                    <div class="ruler">
+                        <div ref="min" :class="labelsClass.min | prefix('ruler__label--')" class="ruler__label">{{ minLabelValue }}</div>
+
+                        <div class="ruler__ticks">
+                            <div v-for="n in ticksCount" :class="tickClass(n) | prefix('ruler__tick--')" :key="n" class="ruler__tick"></div>
+                        </div>
+
+                        <div ref="max" :class="labelsClass.max | prefix('ruler__label--')" class="ruler__label">{{ maxLabelValue }}</div>
+                    </div>
+
+                    <div class="bar__rail">
+                        <div class="rail rail--passive"></div>
+                        <div :class="{'rail--exceeds': limit && value > limitValue}" :style="{width: `${position}px`}" class="rail rail--active"></div>
+                        <div :style="{left: `${position}px`}" class="rail__handle">
+                            <div class="rail__handle-dot"></div>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="slider-helper-text"></div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+import Input from './input.vue'
+
+export default {
+    components: {
+        inputElement: Input,
+    },
+    props: {
+        min: { type: Number, required: true },
```

Comment:
`required: true` is unnecessary since it is the default. Also move all the required props to the top.